mocap_client:
  ros__parameters:
    # Dummy parameters for testing
    dummy_send: false #true. Set it to false to connect to the mocap.
    send_rate: 100
    number_of_bodies: 1 # Same values will be sent for all the bodies
    dummy_x: 0.0
    dummy_y: 0.0
    dummy_z: 1.0
    dummy_qx: 0.0
    dummy_qy: 0.0
    dummy_qz: 0.0
    dummy_qw: 1.0
    # Connection parametrs
    server_address: "192.168.0.98" #"192.168.4.31"
    multicast_address: "239.255.42.99" #default value : 239.255.42.99
    connection_type: 0 #default value: 0. 0 = multicast, 1 = unicast.
    server_command_port: 1510 #default value: 1510
    server_data_port: 1511 #default value: 1511
    pub_topic: "/mocap/rigid_bodies"
    record: false #true. Record flag for Motive software 
    record_file_name: ""
